feature/auth/register/src/main/java/com/loryblu/feature/auth/register/child/ChildRegisterScreen.kt

```diff
@@ -243,14 +242,14 @@ fun ChildRegisterScreen(
 }
 
 
-@Composable
-@Preview
-fun PreviewComposable() {
-    ChildRegisterScreen(
-        viewModel = ChildRegisterViewModel(),
-        navigateToConfirmationScreen = {},
-        onSignUpButtonClicked = {},
-        shouldGoToNextScreen = false,
-        intentForPrivacy = Intent(Intent.ACTION_VIEW),
-    )
-}
+//@Composable
```

Comment:
OK, it will be fix later

Reply:
For save https://jetc.dev/slack/2021-04-17-preview-viewmodel.html